### ISSUE TITLE
Test cases are a bit misleading

### ISSUE THREAD

**Issue by OFFLlNE** [FreeCodeCamp/FreeCodeCamp#11794]
Challenge [Counting Cards](https://www.freecodecamp.com/challenges/counting-cards) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.99 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

![image](https://cloud.githubusercontent.com/assets/8721312/20605022/ee31cff4-b271-11e6-9d1a-b02c72468978.png)

My code:
```javascript

var count = 0;

function cc(card) {
  // Only change code below this line
  switch(card){
    case 2:
    case 3:
    case 4:
    case 5:
    case 6:
      count += 1;
      break;
    case 7:
    case 8:
    case 9:
      count += 0;
      break;
    case 10:
    case 'J':
    case 'Q':
    case 'K':
    case 'A':
      count -= 1;
      break;
  }
  return result();
}
  function result(){
  
  if (count > 0){
    return count + " Bet";
  }
  else{
    return count + " Hold";
  }
  }
/* What I had before
if (count > 0){
    return '"' + count + " Bet" + '"';
  }
  else{
    return '"' + count + " Hold" + '"';
  }
Since the Test cases show that output has to be between "" it made me return it "0 Hold" etc, but actually I did not need ""
*/





// Add/remove calls to test your function.
// Note: Only the last will display
cc(7); cc(8); cc(9);
```

So overall I would suggest removing the "" from the tests

**Comment by Bouncey** [comment]
@OFFLlNE thank you for reporting this issue
-
Can we have a PR to remove the quotes around the expected return in the test cases

**Comment by babattles** [comment]
I would like to try to solve this bug. This will be my first PR.

**Comment by abhisekp** [comment]
Thank you for reporting this issue. This has been fixed in #10225

This is a standard message notifying you that this issue is present in production, but it has been fixed in staging. Because of that, I'm closing this issue. When staging will be pushed to production again, your issue will be resolved.

If you think I'm wrong in closing this issue, please reopen it and add further clarification. Thank you, and happy coding.

closes #11803
